shared/menubar/index.js

```diff
@@ -139,6 +140,17 @@ class Menubar extends Component<void, Props, void> {
     this._closeMenubar()
   }
 
+  _openShell () {
+    if (process.platform === 'win32') {
+      let shellCmd = 'start "Keybase Shell" "%APPDATA%\\Microsoft\\Windows\\Start Menu\\Programs\\Keybase\\Keybase Shell.lnk"'
```

Comment:
@zanderz Is this the preferred way of doing it? what if they remove keybase shell form the start menu?